src/standardTypes.js

```diff
@@ -1,10 +1,15 @@
+const List = ({ args: [tpe] }, { gen }) => `t.list(${gen(tpe)})`;
+
 export default {
   Int: () => 't.Number',
   Float: () => 't.Number',
   String: () => 't.String',
   Boolean: () => 't.Boolean',
   Date: () => 't.String',
   DateTime: () => 't.String',
-  List: ({ args: [tpe] }, { gen }) => `t.list(${gen(tpe)})`,
-  Option: ({ args: [tpe] }, { gen }) => `t.maybe(${gen(tpe)})`
+  List,
+  Option: ({ args: [tpe] }, { gen }) => `t.maybe(${gen(tpe)})`,
+  Set: List,
+  TreeSet: List,
+  Map: ({ args: [tpe] }, { gen }) => `t.dict(t.String, ${gen(tpe)})`
```

Comment:
are you sure about this? `Map` should receive two types as args and the first one should be the key.
In GDSM we override `Map` as:
```
({ args: [k, v] }, { gen }) => t.dict(${gen(k)}, ${gen(v)})
```